src/erldns_handler.erl

```diff
@@ -165,11 +161,12 @@ safe_handle_packet_cache_miss(Message, AuthorityRecords, Host) ->
                 Response ->
                     maybe_cache_packet(Response, Response#dns_message.aa)
             catch
-                Exception:Reason:Stacktrace ->
+                _Exception:Reason:_Stacktrace ->
```

Comment:
We lose visibility over these. Have you considered resurrecting the LOG_ERROR call?

Reply:
For a start unfortunately the log error was commented out, and the handlers were not looking at it, so here I literally just migrated the same behaviour 🥲

Can easily resurrect the error call though, one sec 👌🏽